### DISCUSSION TITLE
Possible to disable registration?

### DISCUSSION THREAD

**Discussion by dghoti** [sysadminsmedia/homebox#271] [Q&A] [answered]
Once the initial (admin) account is set up - can the registration be disabled? (this would stop random signups if main page is hosted externaly)

**Comment by tonyaellie** [comment, marked as answer]
Registrations can be disabled using the HBOX_OPTIONS_ALLOW_REGISTRATION environment variable.

https://homebox.software/en/configure-homebox.html#env-variables-configuration
